todo-app/src/App.js

```diff
@@ -1,13 +1,19 @@
-import React, { Component } from 'react';
+import React from 'react';
+import TodoApp from './TodoApp';
+import {Provider} from 'mobx-react';
+import * as stores from './stores';
 
-class App extends Component {
-  render() {
-    return (
-      <div>
-        Hello World!
-      </div>
-    );
-  }
-}
+window.stores = stores;
```

Comment:
This does not belong in "production" code. If you use something like this for testing, make sure to remove it before committing. Also, the MobX devtools might suit you better. That being said I think you just copy/pasted it from Joost's example.